k8s/vagrant/README.md

```diff
@@ -49,4 +58,69 @@ Just use *vagrant up* to bring up the cluster.
 ubuntu-host:~/k8s-demo$ vagrant up
 ```
 
+### Verify
+
+You should see output similar to this:
+```
+kiran@kmaya:~/k8s-demo$ vagrant status
+Current machine states:
+
+kubemaster-01             running (virtualbox)
+kubeminion-01             running (virtualbox)
+kubeminion-02             running (virtualbox)
+
+This environment represents multiple VMs. The VMs are all listed
+above with their current state. For more information about a specific
+VM, run `vagrant status NAME`.
+kiran@kmaya:~/k8s-demo$ 
+```
+
+## Install kubectl on the host
+
+Please follow the instructions for [install kubectl from binary](https://kubernetes.io/docs/tasks/tools/install-kubectl/#install-kubectl-binary-via-curl).
+
+```
+curl -LO https://storage.googleapis.com/kubernetes-release/release/$(curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt)/bin/linux/amd64/kubectl
+chmod +x ./kubectl
+sudo mv ./kubectl /usr/local/bin/kubectl
+```
+
+### Verify
+
+```
+kiran@kmaya:~/k8s-demo$ kubectl version
+Client Version: version.Info{Major:"1", Minor:"7", GitVersion:"v1.7.5", GitCommit:"17d7182a7ccbb167074be7a87f0a68bd00d58d97", GitTreeState:"clean", BuildDate:"2017-08-31T09:14:02Z", GoVersion:"go1.8.3", Compiler:"gc", Platform:"linux/amd64"}
+The connection to the server localhost:8080 was refused - did you specify the right host or port?
+kiran@kmaya:~/k8s-demo$ 
+```
+
+The connection error is expected. The next step will configure the kubectl to contact the kubernetes cluster. 
+
+## Configure kube-config from the installed cluster
```

Comment:
Configure kube-config from the Installed Cluster

Reply:
done